app/view/home/controlButtons.js

```diff
@@ -449,22 +449,22 @@ getCurrentDisplayModeClass: function() {
 
           classNames: 'sdlGPData',
 
-          contentBinding: 'SDL.SDLController.model.globalProperties.keyboardProperties.autoCompleteText',
+          contentBinding: 'SDL.SDLController.model.globalProperties.keyboardProperties.autoCompleteList',
 
           propertiesData: function() {
+              var str = '';
+              if (SDL.SDLController.model && SDL.SDLController.model.globalProperties.keyboardProperties 
+                && SDL.SDLController.model.globalProperties.keyboardProperties.autoCompleteList) {
+                  var i = 0;
+                  for (i = 0; i < SDL.SDLController.model.globalProperties.keyboardProperties.autoCompleteList.length; i++) {
```

Comment:
Unless you are using this variable outside the for loop, might change the initialization to 
```suggestion
                  for (let i = 0; i < SDL.SDLController.model.globalProperties.keyboardProperties.autoCompleteList.length; i++) {
```
and remove the `var i=0;` in the previous line

Reply:
Addressed in https://github.com/smartdevicelink/sdl_hmi/pull/190/commits/986992bf4ca4a2887c41c23a2a6efa2fefdca6c2